packages/curvenote/src/web/deploy.ts

```diff
@@ -112,20 +114,25 @@ export async function deployContentToCdn(session: ISession, opts?: { ci?: boolea
       size,
     })),
   };
-  const { json: uploadTargets } = await session.post<SiteUploadResponse>(
-    '/sites/upload',
-    uploadRequest,
-  );
 
+  return { files, uploadRequest };
+}
+
+export async function processUpload(
```

Comment:
```suggestion
export async function performFileUploads(
```
or something? Just to be slightly more explicit.